
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>17: Coenzymes &#8212; Organic Chemistry with a Biological Emphasis</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" href="../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Tables" href="Tables.html" />
    <link rel="prev" title="16: Free radical reactions" href="Chapter%2016%20-%20Free%20radical%20reactions.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Organic Chemistry with a Biological Emphasis</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search the docs ..." aria-label="Search the docs ..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        <ul class="list-caption">
 <li class="toctree-l0 has-children">
  <p aria-level="2" class="caption" role="heading">
   <span class="caption-text">
    Organic Chemistry with a Biological Emphasis Part 1
   </span>
  </p>
  <input class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
  <label class="label-parts" for="toctree-checkbox-1">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="nav bd-sidenav">
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%201%20-%20Introduction%20to%20organic%20structure%20and%20bonding%2C%20part%20I.html">
     1: Introduction to organic structure and bonding, part I
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%202%20-%20Introduction%20to%20organic%20structure%20and%20bonding%2C%20part%20II.html">
     2: Introduction to organic structure and bonding, part II
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%203%20-%20Conformation%20and%20stereochemistry.html">
     3: Conformation and stereochemistry
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%204%20-%20Structure%20determination%2C%20part%20I%20-%20Mass%20spectrometry%2C%20infrared%2C%20and%20UV-visible%20spectroscopy.html">
     4: Structure determination, part I: Mass spectrometry, infrared, and UV-visible spectroscopy
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%205%20-%20Structure%20determination%2C%20part%20II%20-%20Nuclear%20magnetic%20resonance%20spectroscopy.html">
     5: Structure determination, part II: Nuclear magnetic resonance spectroscopy
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%206%20-%20Overview%20of%20organic%20reactivity.html">
     6: Overview of organic reactivity
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%207%20-%20Acid-base%20reactions.html">
     7: Acid-base reactions
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Chapter%208%20-%20Nucleophilic%20substitution%20reactions.html">
     8: Nucleophilic substitution reactions
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../OCBE_I/Tables.html">
     Table 1: Some characteristic absorption frequencies in IR spectroscopy
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l0 has-children">
  <p aria-level="2" class="caption" role="heading">
   <span class="caption-text">
    Organic Chemistry with a Biological Emphasis Part 2
   </span>
  </p>
  <input checked="" class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox"/>
  <label class="label-parts" for="toctree-checkbox-2">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="current nav bd-sidenav">
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%209%20-%20Phosphate%20Transfer%20Reactions.html">
     9: Phosphate transfer reactions
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2010%20-%20Nucleophilic%20Addition%20to%20Carbonyl%20Groups.html">
     10: Nucleophilic addition to carbonyl groups
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2011%20-%20Nucleophilic%20acyl%20substitution%20reactions.html">
     11: Nucleophilic acyl substitution reactions
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2012%20-%20Reactions%20at%20the%20%E2%8D%BA-carbon%2C%20part%20I.html">
     12: Reactions at the ⍺-carbon, part I
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Interchapter%20-%20Predicting%20metabolic%20pathways.html">
     Interchapter: Predicting metabolic pathways
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2013%20-%20Reactions%20at%20the%20%E2%8D%BA-carbon%2C%20part%20II.html">
     13: Reactions at the ⍺-carbon, part II
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2014%20-%20Electrophilic%20reactions.html">
     14: Electrophilic reactions
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2015%20-%20Oxidation%20and%20reduction%20reactions.html">
     15: Oxidation and reduction reactions
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Chapter%2016%20-%20Free%20radical%20reactions.html">
     16: Free radical reactions
    </a>
   </li>
   <li class="toctree-l1 current active">
    <a class="current reference internal" href="#">
     17: Coenzymes
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="Tables.html">
     Tables
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l0 has-children">
  <p aria-level="2" class="caption" role="heading">
   <span class="caption-text">
    Solutions
   </span>
  </p>
  <input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox"/>
  <label class="label-parts" for="toctree-checkbox-3">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="nav bd-sidenav">
   <li class="toctree-l1">
    <a class="reference internal" href="../Solutions/Solutions%20to%20in-chapter%20problems.html">
     Solutions to In-Chapter Exercises
    </a>
   </li>
   <li class="toctree-l1">
    <a class="reference internal" href="../Solutions/Solutions%20to%20end-of-chapter%20problems.html">
     Solutions to selected end-of-chapter problems
    </a>
   </li>
  </ul>
 </li>
</ul>
    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Theme by the <a href="https://ebp.jupyterbook.org">Executable Book Project</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../_sources/OCBE_II/Chapter 17 - Coenzymes.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-1-pyridoxal-phosphate-vitamin-b-sub-6-sub">
   Section 17.1: Pyridoxal phosphate (Vitamin B
   <sub>
    6
   </sub>
   )
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-plp-in-the-active-site-the-imine-linkage">
     17.1A: PLP in the active site: the imine linkage
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-plp-dependent-amino-acid-racemization">
     17.1B: PLP-dependent amino acid racemization
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-plp-dependent-decarboxylation">
     17.1C: PLP-dependent decarboxylation
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-plp-dependent-retroaldol-and-retro-claisen-cleavage">
     17.1D: PLP-dependent retroaldol and retro-Claisen cleavage
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#e-plp-dependent-transamination">
     17.1-E: PLP-dependent transamination
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#f-plp-dependent-elimination-and-substitution">
     17.1F: PLP-dependent β-elimination and β-substitution
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#g-plp-dependent-elimination-and-substitution-reactions">
     17.1G: PLP-dependent γ-elimination and γ-substitution reactions
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#h-racemase-to-aldolase-altering-the-course-of-a-plp-reaction">
     17.1H: Racemase to aldolase: altering the course of a PLP reaction
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#i-stereoelectronic-considerations-of-plp-dependent-reactions">
     17.1I: Stereoelectronic considerations of PLP-dependent reactions
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-2-thiamine-diphosphate-vitamin-b-sub-1-sub">
   Section 17.2: Thiamine diphosphate (Vitamin B
   <sub>
    1
   </sub>
   )
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-3-thiamine-diphosphate-lipoamide-and-the-pyruvate-dehydrogenase-reaction">
   Section 17.3: Thiamine diphosphate, lipoamide and the pyruvate dehydrogenase reaction
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-4-folate">
   Section 17.4: Folate
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-active-forms-of-folate">
     17.4A: Active forms of folate
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-formation-of-formyl-thf-and-methylene-thf">
     17.4B: Formation of formyl-THF and methylene-THF
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-single-carbon-transfer-with-formyl-thf">
     17.4C: Single-carbon transfer with formyl-THF
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-single-carbon-transfer-with-methylene-thf">
     17.4D: Single-carbon transfer with methylene-THF
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#key-learning-objectives-for-this-chapter">
   Key learning objectives for this chapter
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#problems">
   Problems
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#pathway-prediction-problems">
     Pathway prediction problems
    </a>
   </li>
  </ul>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>17: Coenzymes</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-1-pyridoxal-phosphate-vitamin-b-sub-6-sub">
   Section 17.1: Pyridoxal phosphate (Vitamin B
   <sub>
    6
   </sub>
   )
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-plp-in-the-active-site-the-imine-linkage">
     17.1A: PLP in the active site: the imine linkage
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-plp-dependent-amino-acid-racemization">
     17.1B: PLP-dependent amino acid racemization
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-plp-dependent-decarboxylation">
     17.1C: PLP-dependent decarboxylation
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-plp-dependent-retroaldol-and-retro-claisen-cleavage">
     17.1D: PLP-dependent retroaldol and retro-Claisen cleavage
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#e-plp-dependent-transamination">
     17.1-E: PLP-dependent transamination
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#f-plp-dependent-elimination-and-substitution">
     17.1F: PLP-dependent β-elimination and β-substitution
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#g-plp-dependent-elimination-and-substitution-reactions">
     17.1G: PLP-dependent γ-elimination and γ-substitution reactions
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#h-racemase-to-aldolase-altering-the-course-of-a-plp-reaction">
     17.1H: Racemase to aldolase: altering the course of a PLP reaction
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#i-stereoelectronic-considerations-of-plp-dependent-reactions">
     17.1I: Stereoelectronic considerations of PLP-dependent reactions
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-2-thiamine-diphosphate-vitamin-b-sub-1-sub">
   Section 17.2: Thiamine diphosphate (Vitamin B
   <sub>
    1
   </sub>
   )
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-3-thiamine-diphosphate-lipoamide-and-the-pyruvate-dehydrogenase-reaction">
   Section 17.3: Thiamine diphosphate, lipoamide and the pyruvate dehydrogenase reaction
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-17-4-folate">
   Section 17.4: Folate
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-active-forms-of-folate">
     17.4A: Active forms of folate
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-formation-of-formyl-thf-and-methylene-thf">
     17.4B: Formation of formyl-THF and methylene-THF
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-single-carbon-transfer-with-formyl-thf">
     17.4C: Single-carbon transfer with formyl-THF
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-single-carbon-transfer-with-methylene-thf">
     17.4D: Single-carbon transfer with methylene-THF
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#key-learning-objectives-for-this-chapter">
   Key learning objectives for this chapter
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#problems">
   Problems
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#pathway-prediction-problems">
     Pathway prediction problems
    </a>
   </li>
  </ul>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="coenzymes">
<h1>17: Coenzymes<a class="headerlink" href="#coenzymes" title="Permalink to this headline">#</a></h1>
<p><img alt="ttps://legionmagazine.com/en/wp-content/uploads/2010/05/VEDayInset8.jpg" src="../_images/image712.jpeg" /></p>
<p>Photo: Taconis Kryn, Library and Archives Canada - PA169941</p>
<p>The old black and white photograph is haunting. A young boy, perhaps 10
or 11 years old, huddles against a wall outside a soup kitchen, his
mouth in an odd twisted shape that could be expressing either pain or
defiance, his eyes staring straight into those of the viewer. Tucked
into his pants, almost like a pistol in a holster, is a metal spoon.</p>
<p>The photograph was taken in the Netherlands in 1945, at the height of
what the Dutch people still refer to simply as “The Hunger Winter”. With
the western part of the country still occupied by the Nazis, the Dutch
resistance government, based in London, had called for a railway strike
with the aim of stopping German troop movements before a planned
airborne invasion by Allied forces. In retaliation, the Germans cut off
all food shipments to cities in the western Netherlands. The Allied
invasion failed to liberate the country, and the winter of 1944-1945
turned out to be bitterly cold. With food supplies dwindling, rations
were cut first to 1000 calories per day, then to 500. People resorted to
eating grass and tulip bulbs just to stay alive. Over 20,000 people died
of starvation before food shipments were restored in the spring.</p>
<p>As tragic as the Hunger Winter was for the Dutch people, some good did
come from it. For medical scientists, the event became a unique ‘social
experiment’: unlike most episodes of famine throughout human history,
the Hunger Winter had a clearly defined beginning, end, and geographic
boundary, and it occurred in a technologically advanced society that
continued to keep thorough records before, during, and after the ordeal.
Scientists knew exactly who suffered from famine and for precisely how
long, and in the years that followed they were able to look at the
long-term effects of famine, particularly on developing embryos.
Researchers found that babies who had been conceived during the famine
were born with a significantly higher incidence of neurological birth
defects such as spina bifida, a condition in which a portion of the
neural tube protrudes from between vertebrae which did not properly fuse
together during fetal development. Later in life, members of this same
cohort of ‘famine babies’ were more likely to be obese, and to suffer
from schizophrenia.</p>
<p>These initial findings spurred interest in further research into the
consequences of prenatal deprivation. In particular, carefully
controlled studies later led to the recognition of the importance of the
vitamin folate in ensuring proper neurological development in early-term
fetuses.</p>
<p><img alt="../_images/image713.png" src="../_images/image713.png" /></p>
<p>fig 1a</p>
<p>Folate - the conjugate base of folic acid - is an organic <strong>coenzyme</strong>:
a helper molecule that binds in the active site of certain enzymes and
plays a critical role in the biochemical reaction being catalyzed.
Recall that we have seen coenzymes at work before: SAM, ATP,
NAD(P)<sup>+</sup> and NAD(P)H, flavin and glutathione are all important
coenzymes with which we are already familiar.</p>
<p>Because prenatal folate deficiency was found to be directly related to
the incidence of spina bifida and other neural tube defects, health
officials in the United States and many other countries changed their
official guidelines to include a specific recommendation that women
begin taking folate supplements as soon as they knew that they are
pregnant, or better yet as soon as they begin trying to become pregnant.
A number of studies conducted during the 1980s and early 1990s
consistently showed that folate supplementation correlated with a 50-70%
reduction in neural tube defects.</p>
<p>The molecular role of folate in prenatal neurological development is not
understood in detail, but most researchers agree that it probably has a
lot to do with DNA biosynthesis. Like *S-*adenosyl methionine (SAM),
folate functions in 1-carbon transfer reactions, including several
critical steps in the nucleic acid biosynthesis pathways. The rapidly
dividing cells of the developing brain of an early term fetus appear to
be especially sensitive to folate deficiency in the mother’s diet:
insufficient folate leads to impaired DNA biosynthesis, which in turn
leads to defects in brain development.</p>
<p>Folate also serves as a 1-carbon donor in the pathway by which SAM is
regenerated after it donates a methyl group. You may recall from the
introduction to chapter 8 that methylation of cytosine bases in DNA by
SAM results in permanent changes to a individual’s genome - this was the
reason why the two ‘identical’ twin sisters in that introductory story
turned out to be, as they grew older, not so identical after all. It is
likely that the folate deprivation that afflicted expectant mothers
during the Dutch Hunger Winter also caused epigenetic changes (in other
words, changes in the extent of DNA methylation) in their developing
fetuses, which decades later manifested in the form of an increased
incidence of conditions such as obesity and schizophrenia. All the more
reason, we now know, to make sure that women get plenty of folate in
their diet early in the first trimester of pregnancy.</p>
<p>In this final chapter, we focus on the organic chemistry of folate,
along with three other coenzymes: pyridoxal phosphate, thiamine
diphosphate, and lipoamide. Humans can synthesize lipoamide, but we
depend on dietary sources for the other three: pyridoxal phosphate is a
form of vitamin B<sub>6</sub>, and thiamine diphosphate is a form of
vitamin B<sub>1.</sub> In a mechanistic sense, there is really nothing
new in this chapter. All of the reaction mechanism types that we will
see are already familiar to us, ranging from nucleophilic substitutions
(chapter 8) to disulfide exchanges (chapter 15). We will soon see,
however, how each coenzyme plays its own specific and crucial role in
assisting enzymes with the catalysis of key reactions of metabolism. We
will begin with pyridoxal phosphate and its various roles in amino acid
metabolism.</p>
<p><u>Additional reading:</u></p>
<p>http://www.naturalhistorymag.com/features/142195/beyond-dna-epigenetics</p>
<section id="section-17-1-pyridoxal-phosphate-vitamin-b-sub-6-sub">
<h2>Section 17.1: Pyridoxal phosphate (Vitamin B<sub>6</sub>)<a class="headerlink" href="#section-17-1-pyridoxal-phosphate-vitamin-b-sub-6-sub" title="Permalink to this headline">#</a></h2>
<p>The coenzyme pyridoxal phosphate (commonly abbreviated PLP) is the
active form of vitamin B<sub>6</sub>, or pyridoxine.</p>
<p><img alt="../_images/image714.png" src="../_images/image714.png" /></p>
<p>fig 1</p>
<p>PLP is required for over 100 different reactions in human metabolism,
primarily in the various amino acid biosynthetic and degradation
pathways. The essential function of PLP is to act as an ‘electron sink’,
stabilizing a negative formal charge that develops on key reaction
intermediates. Some of reactions will be familiar to you from chapter 12
and 13: we will see examples, for instance, of PLP-dependent α-carbon
racemization, as well as aldol- and Claisen-type reactions. Other
reactions will be less familiar: for example, the participation of PLP
allows for decarboxylation of amino acids, a chemical step which would
be highly unlikely without the coenzyme, and PLP is also required for a
very important class of biochemical transformation called
‘transamination’, in which the amino group of an amino acid is
transferred to an acceptor molecule. Before we dive into the reactions
themselves, though, we need to begin by looking at a key preliminary
step that is common to all of the PLP reactions we will see in this
section.</p>
<section id="a-plp-in-the-active-site-the-imine-linkage">
<h3>17.1A: PLP in the active site: the imine linkage<a class="headerlink" href="#a-plp-in-the-active-site-the-imine-linkage" title="Permalink to this headline">#</a></h3>
<p>The common catalytic cycle of a PLP-dependent enzyme begins and ends
with the coenzyme <em>covalently</em> linked to the enzyme’s active site
through an imine linkage between the aldehyde carbon of PLP and the
amine group of a lysine residue (see section 10.5 to review the
mechanism for imine formation). For a PLP-dependent enzyme to become
active, a PLP molecule must first enter the active site of an enzyme and
form an imine link to the lysine. This state is often referred to as an
<strong>external aldimine</strong>.</p>
<p><img alt="../_images/image715.png" src="../_images/image715.png" /></p>
<p>fig 2</p>
<p>The first step of virtually all PLP-dependent reactions is
transimination (section 10.5), as the amino group on the amino acid
substrate displaces the amino group of the enzymatic lysine. This
state - where the coenzyme is covalently linked to the substrate or
product of the reaction - is often referred to as an <strong>internal
aldimine</strong>.</p>
<p><img alt="../_images/image716.png" src="../_images/image716.png" /></p>
<p>fig 3</p>
<p>With the preliminary transimination accomplished, the real PLP chemistry
is ready to start. The versatility of PLP in terms of its ability to
assist with a wide variety of reaction types is illustrated by the
figure below, showing how, depending upon the reaction/enzyme in
question, PLP can assist in the cleavage of any one of the four bonds to
the α-carbon of the amino acid substrate.</p>
<p><img alt="../_images/image7171.png" src="../_images/image7171.png" /></p>
<p>fig 4</p>
<p>Let’s look first at the reaction catalyzed by PLP-dependent alanine
racemase. (EC 5.1.1.1).</p>
</section>
<section id="b-plp-dependent-amino-acid-racemization">
<h3>17.1B: PLP-dependent amino acid racemization<a class="headerlink" href="#b-plp-dependent-amino-acid-racemization" title="Permalink to this headline">#</a></h3>
<p>In section 12.2B we saw an example of a PLP-<em>independent</em> amino acid
racemization reaction, in which the negatively-charged intermediate was
simply the enolate form of a carboxylate:</p>
<p><img alt="../_images/image718.png" src="../_images/image718.png" /></p>
<p>fig 5</p>
<p>Many other amino acid racemase reactions, however, require the
participation of PLP.</p>
<p>Like all other PLP-dependent reactions that we will see in this section,
PLP-dependent amino acid racemization begins with a preliminary step in
which the substrate becomes attached to the coenzyme through a
transimination. Once it is linked to PLP in the active site, the
α-proton of an amino acid substrate is abstracted by an active site base
(step 1 below). The negative charge on the carbanion intermediate can,
of course, be delocalized to the carboxylate group. The PLP coenzyme,
however, provides an expanded network of conjugated π-bonds over which
the electron density can be delocalized all the way down to the PLP
nitrogen. This is what we mean when we say that the job of PLP is to act
as an ‘<strong>electron sink</strong>’: the coenzyme is very efficient at absorbing,
or delocalizing, the excess electron density on the deprotonated
α-carbon of the reaction intermediate. <em>PLP is helping the enzyme to
increase the acidity of the α-hydrogen by stabilizing the conjugate
base</em>. A PLP-stabilized carbanion intermediate is commonly referred to
as a <strong>quinonoid intermediate</strong>. Note that in the overall reaction
equation below, PLP appears below the reaction arrow in brackets,
indicating that it participates in the mechanism but is regenerated as
part of the reaction cycle.</p>
<p>PLP-dependent amino acid racemization:</p>
<p><img alt="../_images/image719.png" src="../_images/image719.png" /></p>
<p>Mechanism:</p>
<p><u>Preliminary step - transimination:</u></p>
<p><img alt="../_images/image720.png" src="../_images/image720.png" /></p>
<p><u>First step - deprotonation:</u></p>
<img alt="../_images/image721.png" src="../_images/image721.png" />
<p>Second step - reprotonation from the opposite side:</p>
<p><img alt="../_images/image722.png" src="../_images/image722.png" /></p>
<p><u>Final step - transimination:</u></p>
<p><img alt="../_images/image723.png" src="../_images/image723.png" /></p>
<p>fig 7</p>
<p>fig 6 fig 6a</p>
<p>Just as in the PLP-independent racemase reactions, reprotonation occurs
on the <em>opposite</em> side of the substrate (step 2), leading to the D-amino
acid product.</p>
<p>All that remains is the final imine exchange which frees the D-amino
acid product and re-attaches the coenzyme to the enzymatic lysine
side-chain, ready to begin another catalytic cycle.</p>
<p><em>To simplify matters, from here on we will not include the preliminary
and final transimination steps in our PLP reaction figures - we will
only show mechanistic steps that occur while the substrate is attached
to the coenzyme (the internal aldimine forms).</em></p>
</section>
<section id="c-plp-dependent-decarboxylation">
<h3>17.1C: PLP-dependent decarboxylation<a class="headerlink" href="#c-plp-dependent-decarboxylation" title="Permalink to this headline">#</a></h3>
<p>In the amino acid racemase reaction above, PLP assisted in breaking the
α-carbon to α-proton bond of the amino acid. Other PLP-dependent enzymes
can catalyze the breaking of the bond between the α-carbon and the
carboxylate carbon by stabilizing the resulting carbanion intermediate:
these are simply decarboxylation reactions.</p>
<p>PLP-depended amino acid decarboxylation:</p>
<p><img alt="../_images/image724.png" src="../_images/image724.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image725.png" src="../_images/image725.png" /></p>
<p>fig 8 fig 9</p>
<p>Notice something very important here: while in racemization reactions
the assistance of PLP can be seen as ‘optional’ (in the sense that some
racemase enzyme use PLP and others do not), the coenzyme is <em>essential</em>
for amino acid decarboxylation steps. Without PLP, there is no way to
stabilize the carbanion intermediate, and decarboxylation is not a
chemically reasonable step.</p>
<p><img alt="../_images/image726.png" src="../_images/image726.png" /></p>
<p>fig 10</p>
<p>One example of a PLP-facilitated decarboxylation reaction is the final
step in the lysine biosynthesis pathway: (EC 4.1.1.20).</p>
<p><img alt="../_images/image727.png" src="../_images/image727.png" /></p>
<p>fig 11</p>
<p>Exercise 17.1: Draw mechanistic arrows for the carbon-carbon
bond-breaking step of the PLP-dependent decarboxylation reaction above.</p>
</section>
<section id="d-plp-dependent-retroaldol-and-retro-claisen-cleavage">
<h3>17.1D: PLP-dependent retroaldol and retro-Claisen cleavage<a class="headerlink" href="#d-plp-dependent-retroaldol-and-retro-claisen-cleavage" title="Permalink to this headline">#</a></h3>
<p><em>(It would be a good idea before reading this section to review
aldol/retro-aldol and Claisen/retro-Claisen reaction mechanisms in
sections 12.3 and 13.3, respectively)</em></p>
<p>So far we have seen PLP playing a role in breaking the bond between the
α-carbon and its α-proton (in the racemization reaction), and the bond
between the α-carbon and carboxylate carbon (in the decarboxylation
reaction).</p>
<p><img alt="../_images/image7171.png" src="../_images/image7171.png" /></p>
<p>fig 4</p>
<p>Other PLP-dependent enzymes catalyze cleavage of the bond between the
α-carbon and the first carbon on the amino acid side chain, otherwise
known as the β-carbon. In the serine degradation pathway, serine is
first converted to glycine by a retro-aldol cleavage reaction. Although
a reasonable mechanism could be proposed without the participation of
PLP, this reaction in fact requires the coenzyme to assist in
stabilization of the negative charge on the carbanion intermediate.</p>
<p>A PLP-dependent retro-aldol cleavage reaction</p>
<p>(serine hydroxymethyl transferase, EC 2.1.2.1)</p>
<p><img alt="../_images/image728.png" src="../_images/image728.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image729.png" src="../_images/image729.png" /></p>
<p>fig 12 fig 13</p>
<p>Note that, in this reaction just as in the racemase reaction described
previously, the key intermediate is a PLP-stabilized carbanion, or
quinonoid.</p>
<p>What happens to the (toxic!) formaldehyde produced in this reaction? We
will see later in this chapter how the serine hydroxymethyltransferase
enzyme goes on to use another coenzyme called tetrahydrofolate to
prevent the formaldehyde from leaving the active site and causing damage
to the cell.</p>
<p>PLP also assists in retro-Claisen cleavage reactions (section 13.3C),
such as this step in the degradation of threonine. (EC 2.3.1.29)</p>
<p>A PLP-dependent retro-Claisen reaction:</p>
<p><img alt="../_images/image730.png" src="../_images/image730.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image731.png" src="../_images/image731.png" /></p>
<p>fig 14 fig 15</p>
<p>Notice how, like the retro-aldol reaction, the bond between the α-carbon
and the β-carbon of the amino acid substrate is broken (in step 1b).</p>
</section>
<section id="e-plp-dependent-transamination">
<h3>17.1-E: PLP-dependent transamination<a class="headerlink" href="#e-plp-dependent-transamination" title="Permalink to this headline">#</a></h3>
<p>One of the most important reaction types in amino acid metabolism is
<strong>transamination</strong>, in which an amino group on a donor molecule (often
an amino acid) is transferred to a ketone or aldehyde acceptor molecule.</p>
<p>A transamination reaction:</p>
<p><img alt="../_images/image732.png" src="../_images/image732.png" /></p>
<p>Transamination phase 1 (transfer of amino group from amino acid
substrate to coenzyme)</p>
<p><img alt="../_images/image7331.png" src="../_images/image7331.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image7341.png" src="../_images/image7341.png" /></p>
<p>fig 16 fig 17 fig17a</p>
<p>Once again, step 1 is abstraction of the α-proton from the PLP-substrate
adduct. However, in a transaminase reaction this initial deprotonation
step is immediately followed by reprotonation at what was originally the
aldehyde carbon of PLP (step 2 above), which results in a new
carbon-nitrogen double bond (in other words, an imine) between the
α-carbon and the nitrogen atom of the original amino acid. The
repositioned imine group is then hydrolyzed (step 3 above), breaking the
carbon-nitrogen bond, transferring the amino group to the coenzyme, and
releasing an α-keto acid.</p>
<p>The coenzyme, which now carries an amine group and is called
<strong>pyridoxamine phospate</strong> (PMP), next transfers the amine group to
α-ketoglutarate (to form glutamate) through a reversal of the whole
process depicted above.</p>
<p>Transamination reaction, phase 2</p>
<p>(transfer of amino group from coenzyme to acceptor molecule)</p>
<img alt="../_images/image7351.png" src="../_images/image7351.png" />
<p>Mechanism:</p>
<p><em>see exercise below</em></p>
<p>fig 18</p>
<p>In a transamination reaction, the PLP coenzyme not only provides an
electron sink, it also serves as a temporary ‘parking place’ for an
amino group as it is transferred from donor to acceptor.</p>
<p><u>Exercise 17.2</u>: Show a complete, step-by-step mechanism for ‘phase
2’ of the transamination reaction above.</p>
<p>Here is an example of a transamination reaction in the arginine
biosynthesis pathway: EC 2.6.1.11</p>
<p><img alt="../_images/image7361.png" src="../_images/image7361.png" /></p>
<p>fig 19</p>
<p><u>Exercise 17.3</u>:</p>
<p>a) Draw arrows for the first mechanistic step of ‘phase 2’ of the above
transaminase reaction.</p>
<p>b) Which carbon on the substrate side of the reaction will eventually
become the α-carbon of arginine?</p>
<p><u>Exercise 17.4</u>: Propose a pathway, with three enzymatic steps, for
the biosynthesis of serine from 3-phosphoglycerate. Include a
generalized (‘-ase’) enzyme name for each step. Glutamate plays a role
in the process as an amino group donor.</p>
<p><img alt="../_images/image7371.png" src="../_images/image7371.png" /></p>
<p>fig 20</p>
</section>
<section id="f-plp-dependent-elimination-and-substitution">
<h3>17.1F: PLP-dependent β-elimination and β-substitution<a class="headerlink" href="#f-plp-dependent-elimination-and-substitution" title="Permalink to this headline">#</a></h3>
<p><em>(Before starting this section, it would be a good idea to review E1cb
β-elimination and conjugate addition reaction mechanisms in chapter
13.4)</em></p>
<p>By now it should be pretty apparent that PLP is a pretty versatile
coenzyme! Two more reaction types in the PLP toolbox are β-elimination
and β-substitution on amino acid substrates.</p>
<p>In a PLP-dependent β-elimination reaction, the coenzyme simply helps to
stabilize the carbanion intermediate of the E1cb mechanism:</p>
<p>A PLP-dependent β-elimination reaction</p>
<p><img alt="../_images/image7381.png" src="../_images/image7381.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image7391.png" src="../_images/image7391.png" /></p>
<p>fig 21 fig 22</p>
<p>Serine dehydratase (EC 4.2.1.13) catalyzes a PLP-dependent β-elimination
in the first step of the serine degradation pathway:</p>
<p><img alt="../_images/image740.png" src="../_images/image740.png" /></p>
<p>fig 23</p>
<p>A β-substitution reaction is simply E1cb elimination followed directly
by the reverse reaction (conjugate addition) with a different
nucleophile (Y in the figure below):</p>
<p>A β-substitution reaction:</p>
<p><img alt="../_images/image741.png" src="../_images/image741.png" /></p>
<p>fig 24</p>
<p>In many bacteria, the synthesis of cysteine from serine includes a
PLP-dependent β-substitution step (EC 2.5.1.47).</p>
<p><img alt="../_images/image742.png" src="../_images/image742.png" /></p>
<p>fig 25</p>
<p><u>Exercise 17.5</u>: Draw a mechanism for the conjugate addition phase
of the reaction above (end with the cysteine-PLP adduct).</p>
</section>
<section id="g-plp-dependent-elimination-and-substitution-reactions">
<h3>17.1G: PLP-dependent γ-elimination and γ-substitution reactions<a class="headerlink" href="#g-plp-dependent-elimination-and-substitution-reactions" title="Permalink to this headline">#</a></h3>
<p>The electron sink capability of PLP allows some enzymes to catalyze
eliminations at the γ-carbon of some amino acid side chains, rather than
at the β-carbon. The secret to understanding the mechanism of a
γ-elimination is that PLP essentially acts as an electron sink <em>twice</em> -
it absorbs the excess electron density from not one but two proton
abstractions.</p>
<p>PLP-dependent γ-elimination</p>
<p><img alt="../_images/image743.png" src="../_images/image743.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image744.png" src="../_images/image744.png" /></p>
<p>fig 26 fig 27</p>
<p>In a familiar first step, the α-proton of the amino acid is abstracted
by an enzymatic base, and the electron density is absorbed by PLP. Next
comes the new part - before anything happens to the electron density
from the first proton abstraction, a second proton, this time from the
β-carbon on the side chain, is abstracted, forming an enamine
intermediate (step 2). The phenolic proton on the pyridoxal ring of PLP
donates a proton to the nitrogen. In step 3, the leaving is expelled and
a new π-bond forms between the β and γ carbons (step 3). This π-bond is
short-lived, however, as the electron density from the first proton
abstraction, which has been ‘stored’ in PLP all this time, flows back up
to protonate the α-carbon (step 4), leaving the γ-elimination product
linked to PLP via the usual imine connection.</p>
<p>An example is the cystathionine γ-lyase reaction in the methionine
degradation pathway (EC 4.4.1.1):</p>
<p><img alt="../_images/image745.png" src="../_images/image745.png" /></p>
<p>fig 28</p>
<p>A related reaction is PLP-dependent γ-substitution, which again is
simply γ-elimination of a leaving group (X in the figure below) followed
directly by the reverse process (a γ-addition) with a different
nucleophile (‘Nu’ in the figure below).</p>
<p>PLP-dependent γ-substitution:</p>
<p><img alt="../_images/image746.png" src="../_images/image746.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image747.png" src="../_images/image747.png" /></p>
<p>fig 29 fig 30</p>
<p>Below is a PLP-dependent γ-substitution reaction in the methionine
degradation pathway (EC 4.2.1.22):</p>
<img alt="../_images/image748.png" src="../_images/image748.png" />
<p>fig 31</p>
</section>
<section id="h-racemase-to-aldolase-altering-the-course-of-a-plp-reaction">
<h3>17.1H: Racemase to aldolase: altering the course of a PLP reaction<a class="headerlink" href="#h-racemase-to-aldolase-altering-the-course-of-a-plp-reaction" title="Permalink to this headline">#</a></h3>
<p>We have seen how PLP-dependent enzymes catalyze a variety of reaction
types - racemization, retroaldol/retro-Claisen cleavage,
transaminination, elimination, and substitution - which, despite their
apparent diversity, are all characterized by formation of a critical
carbanion intermediate which is stabilized by the ‘electron sink’
property of the PLP coenzyme. Given this common mechanistic feature, it
would be reasonable to propose that the active site architecture of
these enzymes might also be quite close. This idea was nicely
illustrated by an experiment in which researchers found that changing a
single active site amino acid of PLP-dependent alanine racemase was
sufficient to turn it into a retro-aldolase (<em>J. Am. Chem. Soc</em>.
<strong>2003</strong>, <em>125</em>, 10158).</p>
<p>In the ‘wild-type’ (natural) alanine racemase reaction, an active site
histidine (red in figure below) deprotonates a neighboring tyrosine
residue (blue), which in turn acts as the catalytic base abstracting the
α-proton of the substrate. When researchers changed the tyrosine to an
alanine (using a technique called ‘site-directed mutagenesis’), and
substituted β-hydroxytyrosine for the alanine substrate, the new
‘mutant’ enzyme catalyzed a retro-aldol reaction.</p>
<p><img alt="../_images/image749.png" src="../_images/image749.png" /></p>
<p>fig 32</p>
<p>Notice what has happened here: the basic histidine, with no tyrosine to
deprotonate because of the mutation, is instead positioned to abstract a
proton from the β-hydroxyl group of the new substrate, setting up a
retroaldol cleavage. That was all it took to change a racemase into a
retroaldolase, because the necessary PLP electron sink system was all
left in place. The researchers predicted correctly that the phenyl ring
of β-hydroxy tyrosine would fit nicely in the space left empty due to
the tyrosine to alanine change in the mutant enzyme’s structure</p>
<p>These results underline the close mechanistic relationship between two
PLP-dependent reactions which, at first glance, appear to be quite
different - and suggest that PLP-dependent racemases and aldolases may
have evolved from a common ‘ancestor’ enzyme.</p>
</section>
<section id="i-stereoelectronic-considerations-of-plp-dependent-reactions">
<h3>17.1I: Stereoelectronic considerations of PLP-dependent reactions<a class="headerlink" href="#i-stereoelectronic-considerations-of-plp-dependent-reactions" title="Permalink to this headline">#</a></h3>
<p>Recall that all PLP-dependent reactions involve the cleavage of one of
the bonds coming from the α-carbon of an amino acid substrate, with the
coenzyme serving as an ‘electron sink’ to stabilize the intermediate
that results. PLP-dependent enzymes accelerate this bond-breaking step
by binding the substrate-PLP adduct in a conformation such that the bond
being broken is close to perpendicular to the plane formed by the
conjugated π system of PLP: this way, as the α-carbon transitions from
sp<sup>3</sup> to sp<sup>2</sup> hybridization, the unhybridized <em>p</em>
orbital is already oriented to overlap with the rest of the conjugated
system. For example, in alanine racemase the first step is cleavage of
the Cα-H bond, so it must be that bond which is positioned
near-perpendicular to the PLP plane:</p>
<img alt="../_images/image750.png" src="../_images/image750.png" />
<p>Likewise, in an amino acid decarboxylase, the Cα-carboxylate bond is
held near-perpendicular to the PLP plane, and in
hydroxymethyltransferase, the Cα-Cβ bond is in the perpendicular
orientation:</p>
<p><img alt="../_images/image751.png" src="../_images/image751.png" /></p>
<p>These are all good examples of how enzymatic catalysis is achieved, in
part, by the ability of the active site to bind the substrate molecule
in a specific conformation which contributes to the lowering of the
activation energy of a key reaction step.</p>
</section>
</section>
<section id="section-17-2-thiamine-diphosphate-vitamin-b-sub-1-sub">
<h2>Section 17.2: Thiamine diphosphate (Vitamin B<sub>1</sub>)<a class="headerlink" href="#section-17-2-thiamine-diphosphate-vitamin-b-sub-1-sub" title="Permalink to this headline">#</a></h2>
<p>Thiamine diphosphate (ThDP, sometimes also abbreviated TPP or ThPP) is a
coenzyme which, like PLP, acts as an electron sink to stabilize key
carbanion intermediates. The most important part of the ThDP molecule
from a catalytic standpoint is its thiazole ring.</p>
<p><img alt="../_images/image752.png" src="../_images/image752.png" /></p>
<p>fig 33</p>
<p>The proton on the carbon between nitrogen and sulfur on the thiazole
ring is weakly acidic, with a pKa of about 18.</p>
<p><img alt="../_images/image753.png" src="../_images/image753.png" /></p>
<p>fig 34</p>
<p>The reason for its acidity lies partly in the ability of the neighboring
sulfur atom to accept, in its open <em>d</em>-orbitals, some of the excess
electron density of the conjugate base. Another reason is that the
positive charge on the nitrogen helps to stabilize the negative charge
on the conjugate base. The deprotonated thiazole is called an <strong>ylide,</strong>
which is a general term for a species with adjacent positively and
negatively charged atoms.</p>
<p>The negatively charged carbon on the ylide form of ThDP is nucleophilic,
and as we shall soon see, the first step of most TPP-dependent reactions
is nucleophilic attack of the ylide carbon on a carbonyl group of the
reaction substrate.</p>
<p>ThDP plays a key role in a variety of reaction types, but the common
theme in all ThDP-dependent reactions is cleavage of a bond adjacent to
the carbonyl carbon of a ketone or aldehyde.</p>
<p>Thiamine diphosphate assists in breaking bonds next to a ketone or
aldehyde:</p>
<p><img alt="../_images/image754.png" src="../_images/image754.png" /></p>
<p>fig 35</p>
<p>Consider this hypothetical decarboxylation step:</p>
<p><img alt="../_images/image755.png" src="../_images/image755.png" /></p>
<p>fig 37</p>
<p>Hopefully, you can quickly recognize that this is <em>not</em> a chemically
reasonable step, because the intermediate species which results from
decarboxylation has a negative charge localized on the ketone carbon - a
very unstable, unlikely intermediate indeed.</p>
<p>(Recall from section 13.1 that decarboxylation steps usually result in
intermediates in which the negative formal charge is delocalized to an
oxygen or nitrogen - in other words, enolates or enamines.)</p>
<p>Now consider, however, a reaction going on in your cells right now,
catalyzed by the enzyme pyruvate decarboxylase (EC 4.1.1.1):</p>
<p><img alt="../_images/image756.png" src="../_images/image756.png" /></p>
<p>fig 36</p>
<p>Somehow, the enzyme manages to accomplish this ‘impossible’
decarboxylation. How does this happen? Here is where the thiamine
diphosphate coenzyme comes in.</p>
<p>A ThDP-dependent decarboxylation reaction (pyruvate decarboxylase):</p>
<p><img alt="../_images/image757.png" src="../_images/image757.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image758.png" src="../_images/image758.png" /></p>
<p>fig 36a fig 38</p>
<p>Upon binding to the enzyme’s active site, ThDP quickly loses a proton.
The nucleophilic ylide carbon then adds to the carbonyl carbon of
pyruvate.</p>
<p>Look carefully at the intermediate that results from step 1 in the
mechanism above. The thiazole ring of ThDP, once it has added to the
carbonyl of pyruvate, provides an ‘electron sink’ to absorb the
electrons from decarboxylation (step 2). Note which bond is breaking in
step 2 - as was mentioned earlier, the common function of ThDP is to
make possible the cleavage of a bond to a carbonyl carbon.</p>
<p>In step 3, the electrons from decarboxylation flow back to abstract a
proton from an acidic group in the active site. All that remains is for
the product to break free of thiamine in step 4.</p>
<p>Thiamine can also assist in decarboxylation-addition reactions:</p>
<p>ThDP-dependent decarboxylation-addition:</p>
<p><img alt="../_images/image759.png" src="../_images/image759.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image7601.png" src="../_images/image7601.png" /></p>
<p>fig 39a fig 39</p>
<p>Here, the electron-rich intermediate formed from the decarboxylation
step (step 2) simply goes on to act as a nucleophile rather than as a
base, adding to the carbonyl group of an aldehyde or ketone (step 3). As
before, the product breaks free of ThDP in step 4.</p>
<p>An example is the first step in the biosynthetic pathway for isoprenoid
compounds in bacteria:</p>
<p><img alt="../_images/image7611.png" src="../_images/image7611.png" /></p>
<p>fig 40</p>
<p>Transketolase, a ThDP-dependent enzyme in the pentose phosphate pathway
of sugar metabolism, catalyzes a carbon-carbon bond break step, followed
by a carbon-carbon bond forming step. The substrates and products are at
similar energy levels, so the reaction is completely reversible.</p>
<p>Transketolase reaction:</p>
<p><img alt="../_images/image7621.png" src="../_images/image7621.png" /></p>
<p>Mechanism:</p>
<p><img alt="../_images/image7631.png" src="../_images/image7631.png" /></p>
<p>Below is an actual example of a transketolase-catalyzed transformation
from the pentose phosphate pathway (shown in Fischer projections, as is
common for sugar structures).</p>
<p><img alt="../_images/image764.png" src="../_images/image764.png" /></p>
<p>fig 43</p>
<p><u>Exercise 17.6</u>: As was mentioned above, the transketolase reaction
is highly reversible. Do you think the same can be said for the
decarboxylation and decarboxylation-addition reactions we saw in this
section? Why or why not?</p>
<p><u>Exercise 17.7</u>: (Challenging!) Propose a mechanism for the
reaction below. Hint: This is a ThDP-facilitated decarboxylation/
Michael addition, followed by E1cb elimination of pyruvate. A Michael
addition is the name for a conjugate addition with a carbon nucleophile.
(<em>J. Mol. Biol</em>. <strong>2010</strong>, <em>401</em>, 253).</p>
<img alt="../_images/image765.png" src="../_images/image765.png" />
<p>fig 44</p>
<p><u>Exercise 17.8:</u> Propose a mechanism for the reaction below.
<em>Hint</em>: the mechanism can be described as a ThDP-facilitated dehydration
step, followed by a tautomerization step, followed by a <em>hydrolytic</em>
expulsion of ThDP (a different kind of ThDP expulsion from what we have
seen so far!)</p>
<p><img alt="../_images/image7661.png" src="../_images/image7661.png" /></p>
<p>fig 44a</p>
</section>
<section id="section-17-3-thiamine-diphosphate-lipoamide-and-the-pyruvate-dehydrogenase-reaction">
<h2>Section 17.3: Thiamine diphosphate, lipoamide and the pyruvate dehydrogenase reaction<a class="headerlink" href="#section-17-3-thiamine-diphosphate-lipoamide-and-the-pyruvate-dehydrogenase-reaction" title="Permalink to this headline">#</a></h2>
<p>The enzyme pyruvate dehydrogenase is one of the most central of all the
enzymes of central metabolism: by converting pyruvate to acetyl-CoA, it
links glycolysis (where glucose is broken down into pyruvate) to the
citric acid cycle, into which carbons enter in the form of acetyl-CoA.
Five ceonzymes are involved: coenzyme A, nicotinamide, thiamine
diphosphate, FAD, and finally lipoamide, one which is new to us at this
point.</p>
<p>Reaction catalyzed by pyruvate dehydrogenase:</p>
<img alt="../_images/image7671.png" src="../_images/image7671.png" />
<p>fig 45</p>
<p>You will learn more about the structure and metabolic role of this
complex and remarkable enzyme in a biochemistry course. Here, we will
focus on the multi-step organic reaction it catalyzes, which we are at
long last equipped to understand.</p>
<p>Looking at the reaction, you should recognize that, first of all, the
pyruvate substrate is being oxidized - it starts out as a ketone, and
ends up as a thioester, losing carbon dioxide in the process.
Ultimately, the oxidizing agent in this reaction is NAD<sup>+</sup>, but
the reduction of NAD<sup>+</sup> is linked to the oxidative
decarboxylation of pyruvate by FAD and a disulfide-containing coenzyme
called <strong>lipoamide</strong>, which is lipoic acid attached by an amide linkage
to a lysine residue on the enzyme.</p>
<img alt="../_images/image7681.png" src="../_images/image7681.png" />
<p>fig 45a</p>
<p>The second thing to notice is that, because the reaction involves
breaking the bond between the ketone carbon and an adjacent carbon,
thiamine diphosphate (ThDP) coenzyme is required. In fact, the first
phase of the reaction (steps 1 and 2 below) is identical to that of
pyruvate <em>decarboxylase</em>, an enzyme we discussed a few pages ago.</p>
<p>The pyruvate decarboxylase reaction mechanism</p>
<p>Phase 1: Decarboxylation of pyruvate</p>
<p><img alt="../_images/image7691.png" src="../_images/image7691.png" /></p>
<p>fig 46</p>
<p>The ThDP-stabilized carbanion then acts as a nucleophile, cleaving the
disulfide bridge of lipoamide (step 3 below). It is in this step that
oxidation of the substrate is actually occurring. After the resulting
thioester product is released from ThDP (step 4 below), it undergoes
transesterification form acetyl-CoA, the product of the reaction.</p>
<p>Phase 2 of the pyruvate decarboxylase reaction mechanism:
lipoamide-mediate oxidation to acetyl-CoA</p>
<p><img alt="../_images/image7701.png" src="../_images/image7701.png" /></p>
<p>fig 47</p>
<p>We are not done yet! In order for the catalytic cycle to be complete,
the reduced dihydrolipoamide must be regenerated back to its oxidized
state through disulfide exchange with a disulfide bond on the enzyme.
The pair of enzymatic cysteines is then oxidized back to disulfide form
by an FAD-dependent reaction.</p>
<p>Phase 3 of the pyruvate decarboxylase reaction mechanism: regeneration
of lipoamide</p>
<p><img alt="../_images/image7711.png" src="../_images/image7711.png" /></p>
<p>fig 48</p>
<p>Finally, FAD is regenerated with concurrent reduction of
NAD<sup>+</sup>:</p>
<p>Phase 4: Regeneration of FADH<sub>2</sub>:</p>
<p><img alt="../_images/image7721.png" src="../_images/image7721.png" /></p>
<p>fig 49</p>
</section>
<section id="section-17-4-folate">
<h2>Section 17.4: Folate<a class="headerlink" href="#section-17-4-folate" title="Permalink to this headline">#</a></h2>
<p>Folate, or vitamin B<sub>9</sub>, is essential for a variety of
important reactions in nucleotide and amino acid metabolism. The
reactive part of folate is the <strong>pterin</strong> ring system, shown in red
below. The conventional atom numbering system for folate is also
indicated.</p>
<p><img alt="../_images/image7731.png" src="../_images/image7731.png" /></p>
<p>fig 50</p>
<section id="a-active-forms-of-folate">
<h3>17.4A: Active forms of folate<a class="headerlink" href="#a-active-forms-of-folate" title="Permalink to this headline">#</a></h3>
<p>Folate is active as a coenzyme in its reduced forms, dihydrofolate and
tetrahydrofolate, which are formed by NADPH-dependent imine
hydrogenation steps (section 15.3).</p>
<p><img alt="../_images/image7741.png" src="../_images/image7741.png" /></p>
<p>fig 51</p>
<p>The metabolic role of folate is to serve as a donor or acceptor in
single-carbon transfer reactions. How is folate different from
<em>S</em>-adenosylmethionine (SAM, section 8.8A) which also serves as a
single-carbon donor? Recall that SAM donates a single carbon in the form
of a methyl group: essentially, the single carbon of SAM is in the
methanol (CH<sub>3</sub>OH) oxidation state, because it has only one
bond to a heteroatom (specifically, to sulfur). (Refer to section 15.1
for a review of oxidation states).</p>
<p><img alt="../_images/image7751.png" src="../_images/image7751.png" /></p>
<p>fig 52a</p>
<p>Folate coenzymes, on the other hand, can carry a single carbon in the
<em>formaldehyde</em> and <em>formate</em> oxidation states, in addition to the
methanol oxidation state. By ‘formaldehyde’ and ‘formate’ oxidation
state, we mean that the carbon has two and three bonds to heteroatoms,
respectively.</p>
<p><img alt="../_images/image7761.png" src="../_images/image7761.png" /></p>
<p>fig 52</p>
<p>Some key reactions in nucleic acid and amino acid metabolic pathways
involve transfer of a single carbon in the formaldehyde or formate
states. However, this could present problems. Formaldehyde by itself is
very toxic: in particular, it tends to spontaneously form unwanted
crosslinks between amine groups (eg. lysine side chains) in proteins.</p>
<p><img alt="../_images/image7771.png" src="../_images/image7771.png" /></p>
<p>fig 53</p>
<p>Free formaldehyde is <em>too reactive</em>, and would cause damage to a cell.
The CH<sub>2</sub>-THF coenzyme is stable in solution, but in the active
site of certain enzymes it is reactive enough to serve as a formaldehyde
donor, as we will see shortly.</p>
<p>Free formate, on the other hand, is a carboxylate, and we know from
chapter 11 that carboxylates are <em>not reactive</em> in acyl substitution
steps. Formate could be activated by phosphorylation, of course, but the
resulting formyl phosphate would be <em>too</em> reactive in many enzyme active
sites. A ‘happy medium’ has been found in which carbons in the formate
oxidation state are carried by folate in the form of f-THF: once again,
the carbon donor is stable in solution, but sufficiently reactive in
certain enzyme active sites to accomplish controlled transfer of a
formate group.</p>
</section>
<section id="b-formation-of-formyl-thf-and-methylene-thf">
<h3>17.4B: Formation of formyl-THF and methylene-THF<a class="headerlink" href="#b-formation-of-formyl-thf-and-methylene-thf" title="Permalink to this headline">#</a></h3>
<p>Formyltetrahydrofolate (f-THF) is formed from THF and a formate molecule
which has been activated by phosphorylation (formyl phosphate, as stated
in the paragraph above, is a high reactive intermediate, but is held
inside the enzyme’s active site for immediate reaction with the incoming
amine group of THF).</p>
<p><img alt="../_images/image778.png" src="../_images/image778.png" /></p>
<p>fig 54</p>
<p>There are two main metabolic routes to CH<sub>2</sub>-THF. One route is
just the last step of the serine hydroxymethyltransferase reaction we
have already seen in section 17.1D: the formaldehyde formed in the
PLP-dependent phase of the reaction stays in the active site, and the
oxygen is displaced by successive attacks from the amine nucleophiles at
the 5 and 10 positions of THF. (Notice the similarity to the
formaldehyde-protein crosslinking reaction shown earlier in this
section.)</p>
<p><img alt="../_images/image779.png" src="../_images/image779.png" /></p>
<p>fig 55</p>
<p>In the second route, f-THF is dehydrated, then the resulting
methenyl-THF intermediate is reduced by NADPH.</p>
<p><img alt="../_images/image7801.png" src="../_images/image7801.png" /></p>
<p>fig 56</p>
<p>Methylene-THF (CH<sub>2</sub>-THF) is reduced to methyl-THF
(CH<sub>3</sub>-THF) in a flavin-dependent reaction. <em>Biochemistry</em>
<strong>2001</strong>, <em>40</em>, 6216</p>
<p><img alt="../_images/image7811.png" src="../_images/image7811.png" /></p>
<p>fig 56a</p>
<p>Recall from the introduction to this chapter that babies who were in the
womb during the Dutch Hunger Winter faced a higher risk, when they
reached adulthood, of conditions such as obesity and schizophrenia, most
likely caused by disruptions in the <em>S</em>-adenosylmethionine-dependent
methylation of their DNA, which was in turn caused by their mothers not
getting enough folate. CH<sub>3</sub>-THF is the source of the methyl
group in methionine, which ultimately becomes the methyl group in SAM.
The methylation of homocysteine to methionine (below) involves a
cobalt-containing coenzyme called cobalamin, but the mechanism for this
reaction is beyond the scope of our discussion. The second reaction
below (formation of SAM) is simply an S<sub>N</sub>2 displacement of the
inorganic triphosphate leaving group on ATP by the nucleophilic sulfur
in methionine.</p>
<p><img alt="../_images/image7821.png" src="../_images/image7821.png" /></p>
<p>fig 56b</p>
</section>
<section id="c-single-carbon-transfer-with-formyl-thf">
<h3>17.4C: Single-carbon transfer with formyl-THF<a class="headerlink" href="#c-single-carbon-transfer-with-formyl-thf" title="Permalink to this headline">#</a></h3>
<p>There are two important f-THF-dependent formylation steps in the
biosynthetic pathways for purine nucleophiles. Both are simply
transamidation reactions: in other words, conversion by the nucleophilic
acyl substitution mechanism (chapter 11) of one amide to another.</p>
<p>Glycinamide ribonucleotide transformylase reaction:</p>
<p><img alt="../_images/image7831.png" src="../_images/image7831.png" /></p>
<p>fig 57</p>
<p>Aminoimidazole carboxamide transformylase reaction:</p>
<p><img alt="../_images/image7841.png" src="../_images/image7841.png" /></p>
<p>fig 58</p>
<p><u>Exercise 17.9</u>: Which of the two transformylase reactions above
would you expect to be more kinetically favorable? Explain your
reasoning.</p>
<p><u>Exercise 17.10</u>: Predict the structure of ‘product X’ in the
reaction below, from the histidine degradation pathway.</p>
<p><img alt="../_images/image7851.png" src="../_images/image7851.png" /></p>
<p>fig 59</p>
</section>
<section id="d-single-carbon-transfer-with-methylene-thf">
<h3>17.4D: Single-carbon transfer with methylene-THF<a class="headerlink" href="#d-single-carbon-transfer-with-methylene-thf" title="Permalink to this headline">#</a></h3>
<p>CH<sub>2</sub>-THF serves as a single-carbon donor in a somewhat
complicated reaction in the biosynthesis of the DNA monomer
doexythymidine monophosphate (dTMP).</p>
<p><img alt="../_images/image7861.png" src="../_images/image7861.png" /></p>
<p>fig 60</p>
<p>The reactions begins with the five-membered ring of CH<sub>2</sub>-THF
breaking apart to create an imine intermediate (step 1 below). The imine
becomes an electrophile in a conjugate addition step (section 13.4,
steps 2-3 below). Note that after step 1, the second ring in the pterin
system of the coenzyme is abbreviated for clarity.</p>
<img alt="../_images/image7871.png" src="../_images/image7871.png" />
<p>fig 61</p>
<p>Next, tetrahydrofolate is eliminated in an E1cb elimination mechanism
(steps 4 and 5). Notice that this is where the single carbon is
transferred from methyl-THF to dUMP.</p>
<img alt="../_images/image7881.png" src="../_images/image7881.png" />
<p>fig 62</p>
<p>The final step in the mechanism is where it gets <em>really</em> interesting: a
hydride ion is transferred from the tetrahydrofolate coenzyme to the
methylene (CH<sub>2</sub>) group on the deoxynucleotide substrate.
Essentially, this is a <em>conjugated</em> S<sub>N</sub>2 step with hydride as
the nucleophile and the active site cysteine as the leaving group.</p>
<p><img alt="../_images/image7891.png" src="../_images/image7891.png" /></p>
<p>fig 63</p>
<p>**<br />
**</p>
</section>
</section>
<section id="key-learning-objectives-for-this-chapter">
<h2>Key learning objectives for this chapter<a class="headerlink" href="#key-learning-objectives-for-this-chapter" title="Permalink to this headline">#</a></h2>
<p><em>After completing this chapter, you should be able to:</em></p>
<p>Understand how pyridoxal phosphate (PLP) acts as an ‘electron sink’ in a
variety of reactions in amino acid metabolism.</p>
<p>Recognize and draw mechanisms for PLP-dependent transformations of the
following types:</p>
<p>racemization</p>
<p>decarboxylation</p>
<p>transamination</p>
<p>retroaldol cleavage</p>
<p>retro-Claisen cleavage</p>
<p>β-elimination</p>
<p>β-substitution</p>
<p>γ-elimination</p>
<p>γ-substitution</p>
<p>Recognize transformations - amino acid decarboxylation and
transamination, for example - in which chemical steps occur that simply
don’t ‘make sense’ unless the electron sink role of PLP is taken into
account.</p>
<p>Understand how the orientation of the substrate in relation to the plane
formed by the conjugated π system of PLP is a major factor in catalysis
of PLP-dependent reactions.</p>
<p>Understand how thiamine diphosphate (ThDP) acts as an ‘electron sink’ in
a variety of reactions in which a bond to a carbonyl carbon is broken,
and how these steps do not ‘make sense’ unless the electron sink role of
ThDP is taken into account.</p>
<p>Recognize transformations for which ThDP is likely required, and be able
to draw reasonable mechanisms for them.</p>
<p>Understand how ThDP acts in tandem with lipoamide, flavin, and
nicotinamide in the reaction catalyzed by pyruvate dehydrogenase.</p>
<p>Recognize folate in its various forms - DHF, THF, f-THF,
CH<sub>2</sub>-THF, and CH<sub>3</sub>-THF - functions in a variety of
one-carbon transfer steps. Be able to recognize the oxidation state of
the carbon being transferred in a folate-depenent step.</p>
</section>
<section id="problems">
<h2>Problems<a class="headerlink" href="#problems" title="Permalink to this headline">#</a></h2>
<p><strong>P17.1:</strong> Here is a chance to test your ability to recognize reactions
catalyzed by enzymes using three coenzymes - thiamine diphosphate,
pyridoxal phosphate, and folate - that we have studied in this chapter.
For each generalized reaction, look carefully at the transformation that
is taking place, and decide which of the three coenzymes is likely to be
required. Then, draw the single mechanistic step by which the bond
identified by an arrow is broken or formed. In the cases where a double
bond is indicated, show the mechanistic step in which the <em>σ bond</em> is
formed. In each case, your drawing should include the structure of the
reactive part of the coenzyme, and should clearly show the role it plays
in catalyzing the mechanistic step you are drawing.</p>
<p>a)</p>
<p><img alt="../_images/image7901.png" src="../_images/image7901.png" /></p>
<p>b)</p>
<p><img alt="../_images/image7911.png" src="../_images/image7911.png" /></p>
<p>c)</p>
<p><img alt="../_images/image7921.png" src="../_images/image7921.png" /></p>
<p>d)</p>
<p><img alt="../_images/image7931.png" src="../_images/image7931.png" /></p>
<p>e)</p>
<img alt="../_images/image794.png" src="../_images/image794.png" />
<p>f)</p>
<p><img alt="../_images/image795.png" src="../_images/image795.png" /></p>
<p>g)</p>
<p><img alt="../_images/image796.png" src="../_images/image796.png" /></p>
<p>h)</p>
<img alt="../_images/image797.png" src="../_images/image797.png" />
<p>i)</p>
<p><img alt="../_images/image7981.png" src="../_images/image7981.png" /></p>
<p>j)</p>
<p><img alt="../_images/image7991.png" src="../_images/image7991.png" /></p>
<p>k)</p>
<p><img alt="../_images/image8001.png" src="../_images/image8001.png" /></p>
<p>l)</p>
<p><img alt="../_images/image8011.png" src="../_images/image8011.png" /></p>
<p>m)</p>
<p><img alt="../_images/image802.png" src="../_images/image802.png" /></p>
<p><strong>P17.2:</strong> The final step in the biosynthesis of the amino acid
tryptophan is a PLP-dependent condensation between serine and indole,
shown below (EC 4.2.1.20). The reaction mechanism involves steps that
are familiar from this chapter, but also incorporates a reaction type we
studied in chapter 14. Propose a mechanism.</p>
<p><img alt="../_images/image803.png" src="../_images/image803.png" /></p>
<p><strong>P17.3:</strong> Draw a reasonable mechanism for the following reaction,
identifying the species denoted by questions marks. (<em>Biochemistry</em>
<strong>2012</strong>, <em>51</em>, 3059)</p>
<p><img alt="../_images/image804.png" src="../_images/image804.png" /></p>
<p><img alt="https://ssl.gstatic.com/ui/v1/icons/mail/images/cleardot.gif" src="../_images/image805.gif" /></p>
<p><strong>P17.4</strong>: Propose a mechanism for the reaction below, which is part of
the anaerobic catabolism of alcohols in some species of bacteria.
(<em>ChemBioChem</em> <strong>2014</strong>, <em>15</em>, 389)</p>
<p><img alt="../_images/image806.png" src="../_images/image806.png" /></p>
<p><strong>P17.5</strong>: Identify cosubstrate A and propose a mechanism for the
reaction shown below, which was reported to occur in the thermophilic
bacterium <em>Thermosporothrix hazakensis</em>. (<em>ChemBioChem</em> <strong>2014</strong> <em>15</em>,
527).</p>
<p><img alt="../_images/image807.png" src="../_images/image807.png" /></p>
<p><strong>P17.6:</strong> Propose a mechanism for each of the reactions below, being
sure to show the role played by the coenzyme (you need to determine
which coenzyme is needed in each case).</p>
<p>a)</p>
<p><img alt="../_images/image808.png" src="../_images/image808.png" /></p>
<p>b) (E.C. 2.2.1.6)</p>
<img alt="../_images/image809.png" src="../_images/image809.png" />
<p>c) <a class="reference external" href="https://www.ncbi.nlm.nih.gov/pubmed/24505011"><em>Angew. Chem. Int. Ed. Engl</em>. <strong>2014</strong>, <em>53</em>,
1943</a>.</p>
<p>http://onlinelibrary.wiley.com/doi/10.1002/anie.201307989/full Fig 1 Orf
R rxn</p>
<p><img alt="../_images/image810.png" src="../_images/image810.png" /></p>
<p><strong>P17.7:</strong> Acetohydroxybutyrate is formed in a coenzyme-dependent
reaction between pyruvate and a 4-carbon compound. What is a likely
second substrate, coenzyme, and by-product (indicated below with a
question mark)?</p>
<p>http://www.sciencedirect.com/science/article/pii/S1367593105001043</p>
<p><img alt="../_images/image811.png" src="../_images/image811.png" /></p>
<p><strong>P17.8</strong>:</p>
<p>a) The ‘benzoin condensation’ reaction was discovered in the 19th
century, and led eventually to a better understanding of ThDP-dependent
reactions in the cell. In a traditional benzoin condensation reaction,
cyanide ion (instead of ThDP) plays the role of electron acceptor.
Enzyme-catalyzed benzoin condensation reactions are also known to occur
in some bacteria*:* Pseudomonas fluorescens, for example, contains an
enzyme that catalyzes the synthesis of (R)-benzoin.</p>
<p><img alt="../_images/image812.png" src="../_images/image812.png" /></p>
<p>a) Draw a mechanism for the enzyme-catalyzed (ThDP-dependent) benzoin
condensation reaction*.*</p>
<p>b) Draw a mechanism for the cyanide-catalyzed benzoin condensation
reaction (non-enzymatic, basic conditions).</p>
<p>c) The following ThDP-reaction was recently reported to be part of the
biosynthetic pathway for clavulanic acid, a compound that inhibits the
action of β-lactamases (β-lactamases are bacterial enzymes that
hydrolyze penicillin-based antibiotic drugs, rendering them
ineffective). As is typical for ThDP-dependent reactions, the first step
is addition of the ylide form of the coenzyme to the substrate carbonyl.
The next steps are (in order): dehydration, tautomerization, elimination
of phosphate, conjugate addition of arginine, and finally hydrolytic
cleavage of the coenzyme-product bond. Draw out a complete mechanism
that corresponds to this description.</p>
<p><img alt="../_images/image813.png" src="../_images/image813.png" /></p>
<p><strong>P17.9:</strong> Practice with PLP-dependent reactions:</p>
<p>a) Propose a mechanism for this reaction, which is part of the
tryptophan degradation pathway (EC 3.7.1.3).</p>
<p><img alt="../_images/image814.png" src="../_images/image814.png" /></p>
<p>b) Propose a mechanism for the final step of the threonine biosynthesis
pathway (EC 4.2.3.1).</p>
<p><img alt="../_images/image815.png" src="../_images/image815.png" /></p>
<p>c) Propose a mechanism for the reaction catalyzed by aspartate
β-decarboxylase (EC 4.1.1.12), which converts aspartate to alanine in a
PLP-dependent reaction.</p>
<p>d) Sphingolipids are a type of membrane lipid found in the membranes of
all eukaryotic cells, and are most abundant in the cells of central the
central nervous system. Synthesis of sphingolipids involves the
PLP-dependent reaction below, catalyzed by serine palmitoyl transferase
(EC 2.3.1.50). Propose a mechanism.</p>
<p><img alt="../_images/image816.png" src="../_images/image816.png" /></p>
<p><strong>P17.10:</strong> As we saw in this chapter, PLP-dependent enzymes usually
catalyze reactions involving amino acid substrates. Here is an
exception, a PLP-dependent β-elimination reaction in the folate
biosynthetic pathway (EC 4.1.3.38). Propose a mechanism for this
reaction.</p>
<p><img alt="../_images/image817.png" src="../_images/image817.png" /></p>
<p><strong>P17.11:</strong> The final step in the degradation pathway for the amino acid
glycine (also known as the ‘glycine cleavage system’) is shown below.
Propose a likely mechanism, given that</p>
<p>evidence suggests that CH<sub>2</sub>NH<sub>2</sub><sup>+</sup> is an
intermediate.</p>
<p><img alt="../_images/image818.png" src="../_images/image818.png" /></p>
<p><strong>P17.12:</strong> As we saw in chapter 15, the usual biochemical role of
NAD<sup>+</sup> is to act as a hydride acceptor in dehydrogenation
reactions. An exception is the reaction catalyzed by the histidine
degradation pathway enzyme urocanase (EC 4.2.1.49).</p>
<p><img alt="../_images/image819.png" src="../_images/image819.png" /></p>
<p>In this reaction, NAD<sup>+</sup> acts as a catalytic, electron-sink
coenzyme - it temporarily accepts electrons from a pi bond in the
substrate, resulting in a covalent substrate-NAD adduct. This allows a
key isomerization step to occur on the substrate through a
protonation-deprotonation mechanism, followed by addition of water,
cleavage of the substrate-NAD adduct to regenerate NAD<sup>+</sup>, and
finally tautomerization to the product. Propose a mechanism that fits
this description, and involves the intermediate below.</p>
<p><img alt="../_images/image820.png" src="../_images/image820.png" /></p>
<section id="pathway-prediction-problems">
<h3>Pathway prediction problems<a class="headerlink" href="#pathway-prediction-problems" title="Permalink to this headline">#</a></h3>
<p>**<br />
**<br />
<strong>P17.13:</strong> Propose a multistep pathway for each of the following
transformations. All involve at least one step requiring PLP, ThDP, or
folate.</p>
<p>a) Below is portion of the biosynthesis of a modified membrane lipid in
<em>Salmonella</em> and other pathogenic bacteria. The modified membrane
confers antibiotic resistance to the bacterium. <em>Biochemistry</em> <strong>2014</strong>,
<em>53</em>, 796</p>
<p><img alt="../_images/image821.png" src="../_images/image821.png" /></p>
<p>b) Below is the biosynthetic pathway for phenethanol in yeast.
Phenethanol, which has a rose scent, is commonly used as a fragrance -
this pathway has been proposed as a potential ‘green’ enzymatic
synthesis to replace the traditional industrial synthesis, which uses
toxic reagents.</p>
<p><img alt="../_images/image822.png" src="../_images/image822.png" /></p>
<p>c) Below is an incomplete pathway diagram for the biosynthesis of the
amino acid lysine, starting from aspartate. Fill in the missing steps
and reactants/coenzymes to complete the diagram. The solid dot and
dashed circle are provided to help you to trace two of the carbons from
substrate to product.</p>
<img alt="../_images/image823.png" src="../_images/image823.png" />
<p>d) Below is the second half of the tryptophan degradation pathway. Fill
in the missing steps and reactants/coenzymes to complete the diagram.</p>
<img alt="../_images/image824.png" src="../_images/image824.png" />
<p>d) Below is an incomplete pathway diagram for the biosynthesis of
inosine monophosphate, a precursor to the nucleotides adenosine
monophosphate (AMP) and guanosine monophosphate (GMP). Fill in missing
steps and reactants/coenzymes to complete the diagram. Note that one
enzymatic step is provided (this is a carboxylation reaction of a type
that we have not studied).</p>
<p><img alt="../_images/image825.png" src="../_images/image825.png" /></p>
<p>e) We begin our study of organic chemistry with a story about a hot
pepper eating contest in Wisconsin (see the introduction the Chapter 1),
and a compound called capsaicin which causes the ‘hot’ in hot peppers.
As our last problem, let’s try to predict some of the key steps in the
biosynthesis of capsaicin.</p>
<p>Phase 1:</p>
<p><img alt="../_images/image826.png" src="../_images/image826.png" /></p>
<p>Phase 2:</p>
<img alt="../_images/image827.png" src="../_images/image827.png" />
</section>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./OCBE_II"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="Chapter%2016%20-%20Free%20radical%20reactions.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">16: Free radical reactions</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="Tables.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Tables</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Tim Soderberg<br/>
  
      &copy; Copyright 2022.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>